main.go

```diff
@@ -68,6 +71,10 @@ var (
 	setupLog                 = ctrl.Log.WithName("setup")
 )
 
+// bootstrapConfigMapName is the name of the ConfigMap that contains cluster-wide
+// configuration for the operator loaded at startup.
+const bootstrapConfigMapName = "operator-bootstrap-config"
```

Comment:
Should we include any notes or warnings that this ConfigMap is not a stable API and should be avoided, or maybe that it is only intended for unsupported enterprise features.

Reply:
I assume we would also want to have this configured by helm when configuring a multi-tenant management cluster.